testdata/cs/expected/pkg/outputtest/repositories.cs

```diff
@@ -2,22 +2,27 @@
 
 using System;
 
-namespace Urlshortener {
+namespace urlshortener.v1 {
```

Comment:
I think you had it correct the first time. Each part of the Namespace is Uppercase. Correct?

This can be done easily in TypeScript with a split, map, join.